packages/fxa-settings/src/lib/utilities.test.ts

```diff
@@ -93,3 +98,20 @@ describe('searchParams', () => {
     expect(Object.keys(params)).toHaveLength(0);
   });
 });
+
+describe('isBase32Crockford', () => {
+  it('is case insensitive', () => {
+    expect(isBase32Crockford('ADF2EGK4HJJ4BTKF')).toBe(true);
+    expect(isBase32Crockford('adf2egk4hjj4btkf')).toBe(true);
+    expect(isBase32Crockford('Adf2eGk4hjJ4BtKF')).toBe(true);
+  });
+
+  it('rejects I, L, O, U', () => {
+    expect(isBase32Crockford('0123456789ABCDEFGHJKMNPQRSTVWXYZ')).toBe(true);
+
+    expect(isBase32Crockford('I')).toBe(false);
+    expect(isBase32Crockford('L')).toBe(false);
+    expect(isBase32Crockford('O')).toBe(false);
+    expect(isBase32Crockford('U')).toBe(false);
```

Comment:
TIL: _"Another alternative design for Base32 is created by Douglas Crockford. It excludes I, L, O to avoid confusion with digits **and U to prevent accidental obscenity**."_

I was totally with you on `I`, `L`, and `O`, but couldn't figure out how `U` fit in.

Reply:
This was news to me as well prior to working on this issue. 😂